app/assets/stylesheets/images.scss

```diff
@@ -1,7 +1,14 @@
+.image-nomination {
+  border-radius: 4rem;
+  border: 0.2rem solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.